main.cpp

```diff
@@ -108,7 +112,19 @@ Thing 4) Coffee shop
     3) serve customer their order
  */
 
-
+/*
+Thing 10) Pet Shelter
+5 properties:
+    1) Dog
+    2) Cat
+    3) Bird
+    4) Lizard
+    5) Snake
```

Comment:
though arrays and inheritance haven't been covered yet, all of these are instances of 'Animal', and will have the same properties, but with different values. 
if arrays had been covered, all of these could live in a single property `std::vector<Animal> animals;` leaving you with 4 more properties to fill in.  
come up with a better object whose sub-objects aren't so similar.
- [ ] revise